README.md

```diff
@@ -132,7 +132,7 @@ Name | Website | Region
 [DigitalOcean](/company-profiles/digitalocean.md) | https://www.digitalocean.com/ | Worldwide
 [Discourse](/company-profiles/discourse.md) | http://www.discourse.org/ | Worldwide
 [DNSimple](/company-profiles/dnsimple.md) | https://dnsimple.com/ | Worldwide
-[Docker](/company-profiles/docker.md) ⚠️️ | https://www.docker.com |
+[Docker](/company-profiles/docker.md) | https://www.docker.com | USA
```

Comment:
Here and in the company profile, we use "Region" to mean "the region from which the company hires remote employees". Since Docker has offices in the US and Europe, I would think they also hire from Europe at least, potentially worldwide. Do you see any info about this on their site?

Reply:
On their website, the only option remote option in the location dropdown is 'Remote, United States'.